### DISCUSSION TITLE
FluentEmail.MailerSend

### DISCUSSION THREAD

**Discussion by marcoatribeiro** [lukencode/FluentEmail#337] [Show and tell]
Hi @lukencode!

I've recently published a package to allow FluentEmail to use [MailerSend ](https://www.mailersend.com/) as sender, as I needed to use this service and found no packages for it.

If you could add a reference to it in your [blog article](https://lukelowrey.com/dotnet-email-guide-2021/), I'd appreciate.

Best regards,
Marco.

**Comment by Jogai** [comment]
Maybe make a PR for the readme under this heading: https://github.com/lukencode/FluentEmail/blob/master/README.markdown?plain=1#L21

**Reply by marcoatribeiro** [reply]
Done. [PR 342](https://github.com/lukencode/FluentEmail/pull/342) created
